includes/compatibility/abstract-wc-data-compatibility.php

```diff
@@ -117,6 +117,25 @@ public static function get_meta($object, $key = '', $single = true, $context = '
             $value     = get_post_meta($object_id, $key, $single);
         }
 
+        if (class_exists('\MyParcelNL\WooCommerce\Includes\Adapter\DeliveryOptionsFromOrderAdapter')) {
```

Comment:
Ik vind dit een beetje vreemd. Graag een duidelijk omschrijving hierboven waarom je dit doet.

Reply:
grappig, dit hoort er helemaal niet in 👀